ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -433,7 +441,20 @@ tests:
       NODE_TUNING: "true"
       TEST_TYPE: conformance-parallel
     workflow: openshift-e2e-libvirt-upi
-- as: ocp-e2e-ovn-remote-libvirt-s390x-heterogeneous
+- as: ocp-e2e-ovn-remote-libvirt-multi-z-z
```

Comment:
ovn is usually after the platform type, but for the remote libvirt jobs, it comes before for some reason